app/src/main/java/com/google/droidjump/DrawableElement.java

```diff
@@ -18,6 +18,11 @@
 
 import android.graphics.Bitmap;
 
+/**
+ * Represents a parent class of all game elements.
+ *
+ * @author maksme@google.com
```

Comment:
Laura, do we actually need authors?

Reply:
Can delete it, just googled how to make java document comments and there were specified author in examples.

Reply:
I would say no, we don't